corehq/apps/groups/models.py

```diff
@@ -181,10 +181,7 @@ def get_case_sharing_groups(cls, domain, wrap=True):
             groups.extend([
                 location.case_sharing_group_object() for location in
                 SQLLocation.objects.filter(domain=domain)
-                # location_type_object is sometimes None
-                # fixes http://manage.dimagi.com/default.asp?158564
-                if getattr(location.couch_location().location_type_object,
-                           'shares_cases', None)
+                if location.couch_location.location_type_object.shares_cases
```

Comment:
Haha, wow yeah that's not the way to do it.  This is a relic of when the `SQLLocation.location_type` was just a string, and this was the simplest way to get the location type object.  I'll update it.